feature-libs/product/configurators/common/src/components/config-overview-form/config-overview-form.component.ts

```diff
@@ -0,0 +1,64 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { Configurator, ConfiguratorCommonsService } from '@spartacus/core';
+import { ConfigRouterExtractorService } from '@spartacus/storefront';
+import { Observable } from 'rxjs';
+import { distinctUntilChanged, filter, switchMap, take } from 'rxjs/operators';
+
+@Component({
+  selector: 'cx-config-overview-form',
+  templateUrl: './config-overview-form.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ConfigOverviewFormComponent {
+  configuration$: Observable<
+    Configurator.Configuration
+  > = this.configRouterExtractorService.extractRouterData().pipe(
+    switchMap((routerData) =>
+      this.configuratorCommonsService.getOrCreateConfiguration(routerData.owner)
+    ),
+    distinctUntilChanged(),
+    switchMap((configuration) =>
+      this.configuratorCommonsService.getConfigurationWithOverview(
+        configuration
+      )
+    ),
+    filter(
+      (configuration) =>
+        configuration.overview !== undefined && configuration.overview !== null
```

Comment:
You can just do `configuration.overview != null` (notice the double equality check, instead of triple equality)